addons/sale/static/src/js/tours/product_configurator_tour_utils.js

```diff
@@ -171,7 +171,7 @@ function assertProductPrice(productName, price) {
         content: `Assert that ${productName} costs ${price}`,
         trigger: `
             ${productSelector(productName)}
-            td.o_sale_product_configurator_price
+            td.o_sale_product_configurator_qty
```

Comment:
Instead this td you can target name=sale_product_configurator_optional_price for more readability :see_no_evil: 
same for other trigger also